src/mbgl/annotation/shape_annotation_impl.hpp

```diff
@@ -6,6 +6,7 @@
 #include <mbgl/util/geo.hpp>
 #include <mbgl/util/geojsonvt/geojsonvt.hpp>
 
+#include <memory>
```

Comment:
Can we move this to `src/mbgl/annotation/shape_annotation_impl.cpp`? It doesn't seem to be use elsewhere in this file.